### ISSUE TITLE
Dependency needs a type convenience method to support aggregator poms

### ISSUE THREAD

**Issue by rktoomey** [sbt/sbt#501]
My project depends on an aggregator pom for Casbah (https://github.com/novus/salat/blob/master/project/SalatBuild.scala).

```
"org.mongodb" %% "casbah" % "2.4.0"
```

When I publish Maven style, the resulting pom doesn't correctly specify that the casbah dependency type is a pom, not a jar (http://search.maven.org/remotecontent?filepath=com/novus/salat-core_2.9.1/1.9.0/salat-core_2.9.1-1.9.0.pom).   

```
<dependency>
  <groupId>org.mongodb</groupId>
  <artifactId>casbah_2.9.1</artifactId>
  <version>2.4.1</version>
  <scope>compile</scope>
</dependency>
```

This works _fine_ for sbt projects, but Maven projects fail to update because they need to have the type explicitly specified:

```
<type>pom</type>
```

See https://groups.google.com/d/topic/simple-build-tool/pox4BwWshtg/discussion for original post to the mailing list.

**Comment by rktoomey** [comment]
Fixed my build by added a pom post processor as suggested by Mark:
https://github.com/novus/salat/commit/1ac5819a18afd52567c1763d273230632c6a9a19

Would really be a nice convenience if I could specify 

```
"org.mongodb" %% "casbah" % "2.4.0" type("pom")
```

And the type would be automatically added by the pom builder.

**Comment by harrah** [comment]
This was fixed in 74c07c64a78447274be47804a6f5f5019c571dab, which is in 0.12.1-RC1.  Use:

``` scala
"org.mongodb" %% "casbah" % "2.4.0" pomOnly()
```

**Comment by rktoomey** [comment]
Thanks so much!
